src/main/java/com/shaft/tools/io/internal/ReportManagerHelper.java

```diff
@@ -51,6 +51,7 @@ public class ReportManagerHelper {
     private static List<List<String>> listOfNewIssuesForFailedTests = new ArrayList<>();
     private static String featureName = "";
     private static Logger logger;
+    private static Logger progressLogger;
```

Comment:
Kindly use the same logger object rather than creating a different one.
creating a different logger config file will add a lot of complexity as we need to bundle it to SHAFT users else it will not work as expected.

Reply:
Regarding this part, I had to make a new logger because the existing one has a newline character at the end of its pattern which will print the progress bar in a newline after each update. Thus, I had to create a new logger with the same pattern as the existing one but only has the newline character removed. 
If there is a way, which I guess there is not, to prevent the existing logger from printing a newline during the progress bar printing process only, then I will be more than glad to implement it.